integration/main.go

```diff
@@ -152,13 +153,64 @@ func createTestNamespace(clientset kubernetes.Interface) string {
 
 func deleteTestNamespace(clientset kubernetes.Interface) {
 	name := opts.namespace
-	if err := clientset.Core().Namespaces().Delete(name, &metav1.DeleteOptions{}); err != nil {
+	if err := clientset.CoreV1().Namespaces().Delete(name, &metav1.DeleteOptions{}); err != nil {
 		glog.Fatalf("failed to delete namespace %q (error: %v)", name, err)
 	}
 	glog.Infof("Namespace %v is deleted", name)
 }
 
 func deployIstioCA(clientset kubernetes.Interface) {
+	// Create Role
+	role := v1alpha1.Role{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "rbac.authorization.k8s.io/v1beta1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "istio-ca-role",
+			Namespace: opts.namespace,
+		},
+		Rules: []v1alpha1.PolicyRule{
+			{
+				Verbs:     []string{"create", "get", "watch", "list", "update"},
+				APIGroups: []string{"core", ""},
```

Comment:
Do we need the empty APIGroup (`""`)?

Reply:
Actually empty APIGroups means "core" when applied via yaml file, but the K8s API does not allow calling with empty field. Also, specifying "core" alone will cause "unknown user" error. I think this may be a bug in K8s. This is the working solution for now.